<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>enquire</class>
 <widget class="QDialog" name="enquire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>请选择查询方式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按姓名查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按学号查询</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>491</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="push">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>80</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>751</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
